_posts/2017-09-25-conference-2017.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: posts
+title: Bazel Conference 2017
+---
+
+Bazel team is pleased to announce our first annual [**Bazel Conference**](https://sites.google.com/corp/bazel.build/conference2017), focused on the needs of our community, with user stories and feedback, migration talks, roadmap, hands-on and break-out tech sessions with Bazel engineers, contributors and users.
+
+**Dates:**     November 6 and 7, 2017
+
+**Location:** Sunnyvale, California 
+
+**We are inspired** by our community engagement in the upcoming conference. We've received a large number of requests for talks and topics for discussion, and have scheduled tech talks by engineers at large and small companies on working with Bazel in wide variety of languages, including Scala, JavaScript and Go. We are looking forward to having 200 engineers from 35 companies across the world share their experience with Bazel.
+
+**We are humbled** by the commitment to make Bazel even better, and are seeing engineers develop advanced features, such as Remote Execution [bazel.buildfarm](https://github.com/bazelbuild/bazel-buildfarm), and sharing migration tips, tricks and tools. You will hear user stories and tips about iOS migration, and moving TensorFlow and Kubernetes to Bazel. We will also discuss as a community what are the different tools out there that could be open sourced. 
```

Comment:
*moving* Tensorflow is a bit weird since they were using Bazel from the beginning. Maybe say and "experience from TensorFlow and Kubernetes team with Bazel".